All EIPIP Meetings/Meeting 049.md

```diff
@@ -0,0 +1,253 @@
+
+**Pooja**: That make total sense. I don't know Liggy are you part of that team uh the fellowship of ethereum magician sorry to put you on the spot I'm just wondering like who would be the best person to reach I know uh Annet and maybe Jimmy but anyone else who are aware of the process behind the scene for fellowship of a ethereum magician. They seem to be stepped away from the microphone maybe for a while but I'll check with the team and we'll try to have some update by the next meeting. And yes one item that I already have shared in the discord channel that now the eipv the eip validator that has been moved to ethereum github repository. So, that's one good thing that we achieved in a while and that also concludes the item listed here anyone else has anything to bring up?
+
+**Micah**: We should probably discuss not many more people in the call but we should probably discuss making Sam and or who is it? Will? tried
+quite a bit of editing lately.
+
+**Pooja**: That's right yep uh yeah I did chat with the light client on this in one of the meetings maybe earlier I don't remember when and we agreed on your suggestion when you recommended that people who feel like being an editor uh should come forward and say that they have been continuously making contribution as you mentioned that we don't have most of the people around do you want me to add it in the next meeting as agenda
+item we can probably have a discussion on this.
+
+**Micah**: Sure
+
+**Pooja**: All right yeah that's a very good point. Now that William and Sam Wilson are there we can probably have them write accesses and it will be helpful to make things moving faster. Okay and um there is one last issue that I can find it on the EIP github that is related to proposal 4337. I don't know if anyone looked into it or not the number is 4783. Maybe, any editor can look into it and leave some comment these are just some uh typos changes, must, should not but I'm understanding if they are under a special bracket there should be a reason behind it.
+
+**Micah**: Let's see which one?
+
+**Pooja**: 4783. Pull Request number is 4783. And the changes are suggested for eip-4337.
+
+**Micah**: Oh yeah. This is this is not any PR. This is just an issue so someone can just comment to on the issue and just tell the user please submit a pr.
+
+**Pooja**: Yeah I was just trying to verify here are these like valid corrections or maybe we can tag author here to jump in right?
+
+Micha: Yeah. So when you create when the person creates an EIP to make these changes or sorry when this person creates a PR to make changes the author will automatically be tagged
+
+**Pooja**: Okay. And then the general recommendation would be like maybe we can comment on this issue asking the person to maybe create a pull request to uh such as these changes to the proposal.
+
+Micha: Yeah this still it might be that the user doesn't know how to create a pull request and so you might just need to walk them through the process.
```

Comment:
```suggestion
**Micha**: Yeah this still it might be that the user doesn't know how to create a pull request and so you might just need to walk them through the process.
```